### PR TITLE
Profiler & Data Quality UI feedback & improvement #7090 part 4

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Work on:-
- display all the tests at table level and provide filter option for the same
- add column which contains redirection linked to the test suite and table
-  show some messages to enable profiler, when profiler is disabled
 [#7090](https://github.com/open-metadata/OpenMetadata/issues/7090)
#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by harshach** [comment]
@ShaileshParmar11 don't put "All Type" -> "All Tests" or put a label "Type" -> "All , Table, Column" , "Status" -> "All, Success, Failed" 
cc @devyani-kaushik

File: openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx [hunk 1]
```diff
@@ -323,6 +323,12 @@ export const generateEntityLink = (fqn: string, includeColumn = false) => {
   }
 };
 
+export const getEntityFqnFromEntityLink = (entityLink: string) => {
+  const link = entityLink.split('>')[0];
+
+  return link.split('::')[2];
```
Comment (vivekratnavel) [inline openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx:329]: We should use antlr in the future

**Review by vivekratnavel — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx:329]: currently is not working as expected, @ayush-shah  is looking into it in 0.12.1

**Comment by ShaileshParmar11** [comment]
> @ShaileshParmar11 don't put "All Type" -> "All Tests" or put a label "Type" -> "All , Table, Column" , "Status" -> "All, Success, Failed" cc @devyani-kaushik

Yes I think putting a label will make it more sense
cc @devyani-kaushik

**Comment by ShaileshParmar11** [comment]
> @ShaileshParmar11 don't put "All Type" -> "All Tests" or put a label "Type" -> "All , Table, Column" , "Status" -> "All, Success, Failed" cc @devyani-kaushik

Hi @harshach i have added label for type and status
image

**Pull request merged**